src/commands/run.ts

```diff
@@ -82,6 +82,13 @@ export function addRun(program: commander.Command): void {
         });
       }
 
+      const stashMessage = 'stashing unstaged changes in order to run hooks properly';
+      const unstashMessage = 'un' + stashMessage;
+      if (type === 'pre-commit') {
+        console.log(chalk.bgCyan(stashMessage));
+        execSync(`git stash push --keep-index --include-untracked -m "MOOKME: ${stashMessage}"`);
+      }
```

Comment:
let's have this as a warning, so in yellow (`chalk.yellow.bold`) + list the stashed files :)
Also a good thing would to only display this if the workspace is not fully staged

Reply:
![Screenshot from 2021-06-04 10-02-24](https://user-images.githubusercontent.com/42519127/120768349-532ac480-c51c-11eb-9276-427c6daf643e.png)

Reply:
Due to some internal configuration of git stash it is not trivial to retrieve the list of untracked files.

I have found [here](https://gist.github.com/imghasemi/015088c966a1ba441b30881816e0d94e) a way to retrieve this list.

Here is now what the output looks like : 
![Screenshot from 2021-06-04 11-30-16](https://user-images.githubusercontent.com/42519127/120780657-9db23e00-c528-11eb-8776-1e06878a70fc.png)

Reply:
And echo the stash command before we execute it : 
`console.log(chalk.bold(\`> git stash push --keep-index --include-untracked -m "MOOKME: ${stashMessage}"\`))`

Reply:
Resolved in a2f1f5f